app/views/content/service_guidance_provider.md

```diff
@@ -232,7 +232,7 @@ Candidates will also be able to use UCAS’s Apply 2 service for this part of th
 
 ### Informing candidates your courses are on Apply
 
-Your courses will automatically be uploaded to [Apply for teacher training](https://www.apply-for-teacher-training.service.gov.uk) as part of our onboarding process.
+Your courses will automatically be uploaded to [Apply for teacher training](https://www.gov.uk/apply-for-teacher-training) as part of our onboarding process.
```

Comment:
This URL is constantized as `GOVUK_START_PAGE_URL` in `govuk_apply_start_page.rb`. Can we use that constant here too for consistency?